README.md

```diff
@@ -40,16 +40,20 @@ configuration file and output directory in development and production.
 
 ## Creating Screenshots
 
-### Downloading the campaign file for a branch
+### 
+The following steps are valid for both, the local dev environment
+and in the production environment.
```

Comment:
The instructions are for production. All the dev commands are described in `DEVELOPMENT.md`